module/metrics.go

```diff
@@ -138,11 +138,17 @@ type ConsensusMetrics interface {
 	// FinishBlockToSeal reports Metrics C4: Block Received by CCL → Block Seal in finalized block
 	FinishBlockToSeal(blockID flow.Identifier)
 
-	// CheckSealingDuration records absolute time for the full sealing check by the consensus match engine
-	CheckSealingDuration(duration time.Duration)
-
 	// EmergencySeal increments the number of seals that were created in emergency mode
 	EmergencySeal()
+
+	// IncreaseOnReceiptDuration increases the number of seconds spent processing receipts
+	IncreaseOnReceiptDuration(duration time.Duration)
+
+	// IncreaseOnApprovalDuration increases the number of seconds spent processing approval
+	IncreaseOnApprovalDuration(duration time.Duration)
+
+	// IncreaseCheckSealingDuration records absolute time for the full sealing check by the consensus match engine
+	IncreaseCheckSealingDuration(duration time.Duration)
```

Comment:
Currently, we implement the metric as a `Counter`. However, I feel this is an implementation detail internal to the metrics component. If we changed the metric type to Histogram, I feel the naming would be off. 

#### Suggestion:
How about we change the names to `OnReceiptProcessingDuration` `OnApprovalProcessingDuration` and `CheckSealingDuration` ?